dhis-2/dhis-api/src/main/java/org/hisp/dhis/route/Route.java

```diff
@@ -48,6 +48,7 @@
 @EqualsAndHashCode(callSuper = true)
 @Accessors(chain = true)
 public class Route extends BaseIdentifiableObject implements MetadataObject {
+  public static final String DEFAULT_RESPONSE_TIMEOUT_SECONDS = "5";
```

Comment:
FYI: You can declare the constant as an `int` and use `""+DEFAULT_RESPONSE_TIMEOUT_SECONDS` in the annotation to turn it into a string.